README.md

```diff
@@ -829,7 +829,7 @@ Other Style Guides
     });
 
     // good
-    [1, 2, 3].map((x) => {
```

Comment:
Actually this is correct - the guide requires that one-arg arrow parens be omitted only for implicit return - and requires that they be included for explicit return.